src/modules/emote_socket_listeners/index.js

```diff
@@ -1,7 +1,32 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import YoutubeHiddenMessage from '../../common/components/YoutubeEphemeralMessage.jsx';
+import {getLiveChat} from '../../utils/youtube.js';
 import {PlatformTypes} from '../../constants.js';
 import {loadModuleForPlatforms} from '../../utils/modules.js';
+import emoteSocketListeners from './emote-socket-listeners.js';
+import twitch from '../../utils/twitch.js';
+
+async function sendEphemeralMessage(message) {
```

Comment:
can we move this to a `youtube.js` util, and call it `youtube.sendChatAdminMessage`?

Reply:
The following error occurs when moved to util, how we can just asynchronously import YoutubeEphermalMessage when needed
```
WARNING in chunk betterttv [mini-css-extract-plugin]
Conflicting order. Following module has been added:
```

Reply:
is there more to this error message than that?

Reply:
```
WARNING in chunk betterttv [mini-css-extract-plugin]
Conflicting order. Following module has been added:
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[2].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[2].use[2]!./node_modules/less-loader/dist/cjs.js??ruleSet[1].rules[2].use[3]!./src/common/components/YoutubeEphemeralMessage.module.css
despite it was not able to fulfill desired ordering with these modules:
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[2].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[2].use[2]!./node_modules/less-loader/dist/cjs.js??ruleSet[1].rules[2].use[3]!./src/modules/emote_menu/components/Icons.module.css
   - couldn't fulfill desired order of chunk group(s) 

webpack 5.75.0 compiled with 1 warning in 1846 ms
```